server/channels/api4/access_control_local.go

```diff
@@ -18,6 +18,8 @@ func (api *API) InitAccessControlPolicyLocal() {
 	api.BaseRoutes.AccessControlPolicies.Handle("/cel/autocomplete/fields", api.APILocal(getFieldsAutocomplete)).Methods(http.MethodGet)
 	api.BaseRoutes.AccessControlPolicies.Handle("/cel/visual_ast", api.APILocal(convertToVisualAST)).Methods(http.MethodPost)
 
+	api.BaseRoutes.AccessControlPolicies.Handle("/sync_job", api.APILocal(createAccessControlSyncJob)).Methods(http.MethodPost)
```

Comment:
I understand this is for mmctl and such, and therefore we don't need a rate limit here. Am I right?